src/User/UserRepository.php

```diff
@@ -26,14 +30,59 @@ public function getSpecialistId(int $userId): int
         return (int) $this->getUserProperty($userId, 'user_specialist_id');
     }
 
-    private function getUserProperty(int $userId, string $property): string
+    public function getNick(int $userId): ?string
+    {
+        return $this->getUserProperty($userId, 'user_nick');
+    }
+
+    private function getUserProperty(int $userId, string $property): ?string
     {
-        return $this->createQueryBuilder()
+        $data = $this->createQueryBuilder()
             ->select($property)
             ->from('users')
             ->where('user_id = :userId')
             ->setParameter('userId', $userId)
             ->execute()
             ->fetchOne();
+        return !is_bool($data) ? $data : null;
```

Comment:
```suggestion
        return $data !== false ? $data : null;
```

Just to be more explicit as PDO returns all properties as string and false in case no entry was found